jenkins.py

```diff
@@ -36,9 +36,12 @@ def get_updated_tools(repos):
         tags = {str(x).split('--')[1] for x in tags if '--' in x}
         # Fetch last commit hash for tool using `git log`
         p = subprocess.Popen(['git', 'log', '--pretty=oneline', '-n', '1', '--', tool], stdout=subprocess.PIPE)
-        commit, comment = p.stdout.read().split(" ", 1)
-        if commit not in tags:
-            updated_tools.add(tool)
+        try:
+            commit, comment = p.stdout.read().split(" ", 1)
+            if commit not in tags:
+                updated_tools.add(tool)
+        except ValueError as e:
+            pass
```

Comment:
While we're at it, the `git` status code isn't being checked and there is no guarantee that you are fully consuming `git`'s output. Either use `check_output` or `p.communicate()` in conjunction with `p.returncode`.